recipes/pandasgui/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pandasgui" %}
+{% set version = "0.2.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pandasgui-{{ version }}.tar.gz
+  sha256: a41ee7ca07a5a6d58b050209366e1a4873f13cc4ca8298fad9d81773f2a9e193
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+  host:
+    - pip
+    - python
+    - python-pyqt5.qtwebkit  # [linux]
+    - setuptools-git
+    - xorg-libxfixes  # [linux]
+  run:
+    - appdirs
+    - dacite
+    - ipython
+    - pandas
+    - plotly
+    - pynput
+    - pyqt 5
+    - python
+
+test:
+  imports:
+    - pandasgui
+    - pandasgui.widgets
+  commands:
+    - pip check
+  requires:
```

Comment:
Would add the same CDTs from `build` above here.

Reply:
@jakirkham @ocefpaf
Done, but without success.

Reply:
Any advice on how to proceed?

Reply:
@conda-forge/staged-recipes

Reply:
Sorry. Out of ideas. I can try a few things later this week but I cannot promise I'll have time for it.

Reply:
That would be very helpful, thanks!